src/utils/resolve-project-base-path.ts

```diff
@@ -3,11 +3,27 @@ import { cosmiconfigSync } from 'cosmiconfig';
 
 import { ProjectType } from '../config';
 
+import { isObject, isString } from './validators.utils';
+
 const angularConfig = ['angular.json', '.angular.json'];
 const workspaceConfig = ['workspace.json'];
+const projectConfig = ['project.json'];
+const sourceRootDefault = 'src';
+
+function searchConfig(searchPlaces: string[], searchFrom = '') {
+  return cosmiconfigSync('', { searchPlaces }).search(searchFrom)?.config;
+}
 
-function searchConfig(searchPlaces: string[]) {
-  return cosmiconfigSync('', { searchPlaces }).search()?.config;
+function defaultWithError(places: string): {
```

Comment:
I think it's best if this function only logs the error and the config is explicitly returned from the function.